src/imfilter_cuda.jl

```diff
@@ -0,0 +1,43 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENCE in the project root.
+# ------------------------------------------------------------------
+
+function imfilter_resource(resource::CUDALibs, img, kern)
+
+  sizeall = size(img) .+ size(kern) .- 1
+
+  imggpu = nothing
+  kerngpu = nothing
+
+  n = ndims(img)
+  t = eltype(img)
```

Comment:
```suggestion
  T = eltype(img)
```

Types are usually written in capital letter.

Reply:
Thanks.  I was in doubt about this.